From 33c08215fd1db3b6095008907d8095eb61bd3bcd Mon Sep 17 00:00:00 2001
From: Ken Sharp <ken.sharp@artifex.com>
Date: Fri, 10 Jun 2016 09:49:02 +0100
Subject: [PATCH] Fix .locksafe

Apparently we need to .forceput the definition of getenve into
systemdict, at least when running GSView 5.0.

Discovered when trying to investigate a customer bug report using
GSView 5.
---
 Resource/Init/gs_init.ps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Resource/Init/gs_init.ps b/Resource/Init/gs_init.ps
index a3b7cdd..9b23d98 100644
--- a/Resource/Init/gs_init.ps
+++ b/Resource/Init/gs_init.ps
@@ -2025,7 +2025,7 @@ readonly def
       /LockFilePermissions true
     >> setuserparams
   }
-  systemdict /getenv {pop //false} put
+  systemdict /getenv {pop //false} .forceput
   if
   % setpagedevice has the side effect of clearing the page, but
   % we will just document that. Using setpagedevice keeps the device
-- 
2.9.3

